.github/benchmark-files/bench.php

```diff
@@ -0,0 +1,353 @@
+<?php
+if (function_exists("date_default_timezone_set")) {
```

Comment:
This file has been copied from the php-src project. It is just a php process that runs some fairly standard algorithms and which serves as an artificial benchmark. I removed a couple of functions which were taking long to run and which made these benchmarks too slow